tests/Test/AbstractFixerTestCase.php

```diff
@@ -703,6 +706,34 @@ private static function assertValidDescription(string $fixerName, string $descri
         self::assertFalse(strpos($descriptionType, '``'), \sprintf('[%s] The %s must no contain sequential backticks.', $fixerName, $descriptionType));
     }
 
+    private static function assertOptionDefault(FixerOptionInterface $option, FixerInterface $fixer): void
+    {
+        if (!$option->hasDefault()) {
+            return;
```

Comment:
if we claim it's assertion method, it should execute assertion in all paths

Reply:
what would you prefer?
1. making the function dynamic to call `addToAssertionCount` (which is a dynamic method)
2. `self::assertTru(true)` 😁 
3. wrapping everything in a huge assert

Reply:
assertion without assert count increased is weird. please increase the count (pick the best option you believe)